packages/components/.storybook/webpack.config.js

```diff
@@ -0,0 +1,21 @@
+const path = require( 'path' );
+
+module.exports = ( { config } ) => {
+	config.module.rules.push( {
+		test: /\.tsx?$/,
+		include: path.resolve( __dirname, '../src' ),
```

Comment:
nit: `path.join(__dirname, '../src')` does the same and arguably is more correct.

Reply:
Good call, done in bef964f1d8b76dc3c655ab5aa0eb977af62e0da0